fly.toml

```diff
@@ -1,43 +1,11 @@
 app = "security-vulnerability-examples-next-js-postgres"
-kill_signal = "SIGINT"
-kill_timeout = 5
-processes = []
-
-# Apps without volume: Comment out the [mounts] configuration below
-# [mounts]
-#  source = "postgres"
-#  destination = "/postgres-volume"
```

Comment:
let's document (in PR diff comments) why each of these different things was removed, and why it's not necessary anymore

Reply:
[20d384f](https://github.com/upleveled/security-vulnerability-examples-next-js-postgres/pull/85/commits/20d384f12b91d6298ca3dc74bf3b022fff863063)